tests/mocks/function-coauthors.php

```diff
@@ -41,3 +41,100 @@ function coauthors( $between = ', ', $betweenLast = ' and ', $before = '', $afte
 
 	echo $output;
 }
+
+/**
+ * A mock for the coauthors_posts_links function from Co-Authors Plus.
+ *
+ * @param string $between Delimiter that should appear between the co-authors
+ * @param string $betweenLast Delimiter that should appear between the last two co-authors
+ * @param string $before What should appear before the presentation of co-authors
+ * @param string $after What should appear after the presentation of co-authors
+ * @param bool   $echo Whether the co-authors should be echoed or returned. Defaults to true.
+ */
+function coauthors_posts_links( $between = null, $betweenLast = null, $before = null, $after = null, $echo = true ) {
+	// To use this function, put display names in a global array called $apple_news_coauthors.
+	global $apple_news_coauthors;
+
+	// Bail if we don't have coauthors.
+	if ( empty( $apple_news_coauthors ) ) {
+		return '';
+	}
+
+	$output = [];
+
+	foreach ( $apple_news_coauthors as $author ) {
+		// Get author data.
+		$author = get_user_by( 'id', $author );
+
+		$args        = array(
+			'before_html' => '',
+			'href'        => get_author_posts_url( $author->ID, $author->user_nicename ),
+			'rel'         => 'author',
+			'title'       => sprintf( __( 'Posts by %s', 'co-authors-plus' ), $author->display_name ),
+			'class'       => 'author url fn',
+			'text'        => $author->display_name,
+			'after_html'  => '',
+		);
```

Comment:
Use square brackets for arrays:
```suggestion
		$args = [
			'before_html' => '',
			'href'        => get_author_posts_url( $author->ID, $author->user_nicename ),
			'rel'         => 'author',
			'title'       => sprintf( __( 'Posts by %s', 'co-authors-plus' ), $author->display_name ),
			'class'       => 'author url fn',
			'text'        => $author->display_name,
			'after_html'  => '',
		];
```